config/clients/go/template/client/client.mustache

```diff
@@ -71,6 +72,15 @@ func NewSdkClient(cfg *ClientConfiguration) (*{{appShortName}}Client, error) {
 	clientConfig := newClientConfiguration(apiConfiguration)
 	clientConfig.AuthorizationModelId = cfg.AuthorizationModelId
 
+	// validate the store id and model id if they are defined
+	if cfg.StoreId != "" && !utils.IsWellFormedUlidString(cfg.StoreId) {
```

Comment:
Instead of validating store id here, call `apiConfiguration.ValidateConfig()` which will handle the validation of all the common parts

Reply:
done